### ISSUE TITLE
test for unlabeled data

### ISSUE THREAD

**Issue by mujtabahasan** [facebook/C3D#117]
Hi,
The current examples on finetuning has testing scripts which expects labels of the videos to compute accuracy. How can it be avoided to allow test on unlabeled real data. Also, the output of test is not stored in any file. Can it be given as an argument during test time?
Thanks,
Mujtaba

**Comment by dutran** [comment]
@mujtabahasan I don't really understand your question. Why do we need to test when we don't have the true labels?

**Comment by mujtabahasan** [comment]
because we have true labels only for the validation set and the labels of test set is not provided. It has to be estimated for any given new video. Also, how to save the prediction labels or the prob layer of the test net.

**Comment by dutran** [comment]
@mujtabahasan you can use extract_image_feature to extract prob and compute your predictions from prob (in matlab or python).

**Comment by mujtabahasan** [comment]
ok. thanks. can i get the prediction in prob in a text format, that is a probability distribution over all labels
